pom.xml

```diff
@@ -53,6 +53,12 @@
         <artifactId>guava</artifactId>
         <version>18.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>5.2.0</version>
+        <scope>test</scope>
```

Comment:
There shouldn't be a scope in `dependencyManagement` dependencies.

Reply:
Removed the scope from the dependencyManagement in this and other pom files as well.